sdk/src/activity.rs

```diff
@@ -525,25 +539,53 @@ impl ActivityBuilder {
         self
     }
 
+    /// Adds up to two buttons with a label and a link other users can click on
+    ///
+    /// Overwrites any secrets already set in the activity.
+    pub fn button(mut self, button: Button) -> Self {
+        let button = ButtonKind::Link(button);
+        match &mut self.inner.activity {
+            Some(Activity {
+                buttons_or_secrets, ..
+            }) => match buttons_or_secrets {
+                Some(ButtonsOrSecrets::Buttons { buttons }) => buttons.push(button),
```

Comment:
What happens if the user adds more then 2 buttons?

Reply:
discord responds with an error
`Err(Discord(Api(InvalidCommand { reason: "child \"activity\" fails because [child \"buttons\" fails because [\"buttons\" must contain less than or equal to 2 items]]" })))`

I wasn't sure if this should be restricted by the library and how to handle that error.

Reply:
I think it's fine, changing it to use a Result would break the builder pattern so I think it's fine to keep this as it is since it's pointed out in the doc comment that there is a max of 2.